client/.eslintrc.json

```diff
@@ -4,7 +4,8 @@
     "../.eslintrc.json",
     "plugin:import/typescript",
     "plugin:react/recommended",
-    "plugin:react-hooks/recommended"
+    "plugin:react-hooks/recommended",
+    "plugin:cypress/recommended"
```

Comment:
Could scope this to just the `/cypress` dir by adding a `/cypress/.eslintrc.json` file that extends with this plugin.